schemas/org.gnome.shell.extensions.noannoyance.gschema.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<schemalist>
+
+  <schema id="org.gnome.shell.extensions.noannoyance" path="/org/gnome/shell/extensions/noannoyance/">
+    <key type="b" name="enable-blacklist">
+      <default>true</default>
+      <summary>Enable blacklist.</summary>
+      <description>If set, it will prevent defined applications (by wm_class) from stealing focus.</description>
```

Comment:
it would be great to unify the (user facing) strings for WM class (you currently use WM class, WM_CLASS, wm_class).